test/test_evaluator.cpp

```diff
@@ -367,27 +367,47 @@ TEST_F(ExprEvalTest, Array) {
   checkIntValue(evaluator, bb, 5L);
 }
 
-TEST_F(ExprEvalTest, Tensor) {
+TEST_F(ExprEvalTest, TensorEagerExecution) {
+  Fusion fusion;
+  FusionGuard fg(&fusion);
+
+  TensorView* tv0 = makeSymbolicTensor(2);
+  TensorView* tv1 = makeSymbolicTensor(2);
+  auto tv2 = add(tv0, tv1);
+
+  at::Tensor a = at::rand({6, 128});
+  at::Tensor b = at::rand({6, 128});
+
+  ExpressionEvaluator evaluator;
+  evaluator.bind(tv0, a);
+  evaluator.bind(tv1, b);
+
+  EXPECT_TRUE(at::allclose(evaluator.evaluate(tv2).as<at::Tensor>(), a + b));
```

Comment:
Yes, exactly. Your understanding is correct.

Reply:
OK, so that means that `Expr` subclasses must implement `evaluate` with aten tensors. If a `Expr` subclass does not have an implementation, it will just fail at https://github.com/NVIDIA/Fuser/blob/main/csrc/ir/base_nodes.cpp#L522, correct?

Reply:
Yes